crates/burn-jit/src/kernel/mask/mask_where.rs

```diff
@@ -88,22 +92,23 @@ fn mask_where_readonly<R: JitRuntime, EI: JitElement, EM: JitElement>(
 
     let cube_dim = CubeDim::default();
     let cube_count = calculate_cube_count_elemwise(input.shape.num_elements(), cube_dim);
+    let vectorization = max_vectorization(&input);
 
-    mask_where_readonly_kernel::launch::<EI, R>(
+    mask_where_readonly_kernel::launch::<EI, EM, R>(
         &input.client,
         cube_count,
         cube_dim,
-        input.as_tensor_arg::<EI>(1),
-        mask.as_tensor_arg::<EM>(1),
-        value.as_tensor_arg::<EI>(1),
-        output.as_tensor_arg::<EI>(1),
+        input.as_tensor_arg::<EI>(vectorization),
+        mask.as_tensor_arg::<EM>(vectorization),
+        value.as_tensor_arg::<EI>(vectorization),
+        output.as_tensor_arg::<EI>(vectorization),
```

Comment:
Does it work?

Reply:
Yes, CUDA splits it into `vectorization` ternaries and both wgsl and SPIR-V support passing a Boolean vector to mix the true and false vector based on each value in the bool vector. All mask tests were passing too.